### ISSUE TITLE
[Suggestion] Preferences? [18+]

### ISSUE THREAD

**Issue by nohumanallow3d** [Monika-After-Story/MonikaModDev#2991]
I think it would be nice to add things about sexual preference to monika's dialogue. Perhaps this may not be a good idea since monika afterstory is not supposed to be 18+? I dont really know but I think this would be a good idea to add.

**Comment by Unknownpony** [comment]
Throwing my thoughts here:
Actually it depends what you mean. I think adult topics should be added in future at some point. Most people using MAS are over 18+ and relationship (as MAS is kind of virtual gf) have to get to the point of 18+ topics. It doesn't actually need to be in pervert way as I imagine most people think when such suggestion is made hehe

**Comment by Delcos** [comment]
I think it would be a refreshing direction.

**Comment by DbzGOD2000** [comment]
I actually wouldn't mind seeing some 18+ topics like what Delcos said how it can be refreshing.

**Comment by TheGlader2** [comment]
If the mod takes a gross direction like this, I'm out entirely, sensitive mode or not, it wouldn't matter if I was working on something at the time. I also don't think it would be good for some people to be forced to either play with this content or have to avoid it. Cause some people can't control themselves and it might not be good for their mental health.

Not everyone 18+ would want this, or should have it.
And as a kid I would be able to figure this mod out, it's easy peasy to install. Do you guys not remember being kids? I think making an email is harder. You just drag and drop, and finding it, all you got to do is have it recommended to you on youtube, or look up DDLC mods.
Don't fight with me about it, just give me a thumbs up or down if you want to leave an opinion. This is pretty black and white stuff. Either you agree or you don't.

**Comment by Bam8647** [comment]
This definitely should be given a poll.

There are many who enjoy her being pure and innocent while others may be _dying_ to see a different side to her personality eventually. 

This could also tie into **"Player's influence over Monika"** where the player can decide if they like her delving into these types of conversations or not.

**Comment by Dregdael** [comment]
Yeah, I don't know, maybe a bit here and there, something not explicit I suppose would be a good compromise

**Comment by TheGlader2** [comment]
@Bam8647 The problem with this, is that you and Monika are forced into a romantic relationship in the mod, so it wouldn't make sense to not do these things if given the choice. (that is, if this mod plans on being consistent in anyway)

If it's going to be an option, then Monika shouldn't kill herself if you say you don't love her in the beginning. It shouldn't be reversible either without reinstall, unless it's a one time thing once the update hits. That could have some cool dialogue.

And her being innocent and pure is up for debate like everywhere. It's kind of a can of worms topic.

**Comment by MaximumRustler** [comment]
Personally, I don't think any more topics than what we already have should be added about this subject right now. There isn't much info on what Monika's preferences are besides the topic where she says something like ''I'm a pure and innocent highschool girl.'', I think that's enough. As for the player's preferences, things like this should be left out till the day she becomes real or is somewhat of a proper A.I., otherwise it's pointless fanservice in my opinion. I'd rather have other, more interesting topics or character developing topics instead of something like this.

**Comment by MaximumRustler** [comment]
Yep, a sentient Monika is my dream. lol

**Comment by Hapilone** [comment]
I'm cool with it as long as it's not done in a truly perverted way that would go against her character.

**Comment by TheGlader2** [comment]
@Foxinsunglasses I get that a lot, I don't want to get into it, but one of the first things Monika says in the mod when talking about her old character file is: That wasn't really me anyway, all it was was a bunch of random extra code that
"It looks like the extra code made me connect to you in a special way."
"Or maybe it's our eternal love~"

It to me originally looked like an opportunity for a character arc, and got me excited at the idea of Monika questioning her own existence and why she was designed to fall in love with the player and what she wants to do about it . . . I don't think that's the way the devs want to take the mod, but that's what I thought upon seeing it. I would really like to see some existential emotional internal conflicts for Monika, there is much opportunity for it here to develop her character. But that's what I've hoped in the past, I've kind of resigned on those ideas though.

**Comment by Torwald39** [comment]
I am against sexual pursuit. I want to keep my pure Monika.

However, I would like new serious conversations.

**Comment by Storylover-Vodhr** [comment]
Eh, I'd very much rather us work towards Monika becoming more of a fleshed out person before we start working towards stuff like that. 

I mean, she doesn't even know our preferences for most things, like if we like pets, or our favorite food or season. Nor does she have many topics delving into what she herself would like to do, beyond getting together with the player. 

Heck, we don't even have many ways of true interaction with her yet. 

I say we figure out who she is first, before adding anything of that nature.

**Comment by nohumanallow3d** [comment]
> I am against sexual pursuit. I want to keep my pure Monika.
> 
> However, I would like new serious conversations.

**Comment by nohumanallow3d** [comment]
> If the mod takes a gross direction like this, I'm out entirely, sensitive mode or not, it wouldn't matter if I was working on something at the time. I also don't think it would be good for some people to be forced to either play with this content or have to avoid it. Cause some people can't control themselves and it might not be good for their mental health.
> 
> Not everyone 18+ would want this, or should have it.
> And as a kid I would be able to figure this mod out, it's easy peasy to install. Do you guys not remember being kids? I think making an email is harder. You just drag and drop, and finding it, all you got to do is have it recommended to you on youtube, or look up DDLC mods.
> Don't fight with me about it, just give me a thumbs up or down if you want to leave an opinion. This is pretty black and white stuff. Either you agree or you don't.

its not exactly 'gross' as you put it its just part of life and obv devs wont put in extremely sexual hardcore stuff in the mod cuz thats literally porn I mean just mention sayyyy Whether u are dominant or submissive. Monika herself should be openly talk about sexuality to the player since obv ur in a relationship with her. I'd say it would be fitting for her to ask or say this kind of stuff when u have told her 'I Love you!' or 'I Love the Way You Look!'.

**Comment by Unknownpony** [comment]
I think some requirments should be in order too. Like at least year or two dating with Monika before she start talking about more spicy topics. Also obviously no porn... (Let's be honest. I am sure some submods would be made for that eventually when MAS API will be made)

**Comment by nohumanallow3d** [comment]
> I think some requirments should be in order too. Like at least year or two dating with Monika before she start talking about more spicy topics. Also obviously no porn... (Let's be honest. I am sure some submods would be made for that eventually when MAS API will be made)

YO WHAT A yeeeaaaaar damn man u be going reallll slow not to mention u can confess loving monika really early bro id say a month or  two since thats what most relationships are irl

**Comment by MaximumRustler** [comment]
Well I don't agree with the way most people handle relationships to tell you the truth. In my opinion, the way most people handle them is stupid and why they fail in the end. So you saying that ''that's what most relationships are irl'' is a really bad example/standard to go by, and Monika's character is quite different compared to ''the average person'', part of the reason people are serious about loving her. So from a character perspective it makes no sense, at least that early. And finally judging by the way you talk, I'd probably disagree with most of your suggestions, you sound very immature, exactly what this mod doesn't need. Excuse me if that seems too harsh.

**Comment by MaximumRustler** [comment]
To add more to what I said.

> you might end up getting a gf irl and just thinking she's pure but she isnt

Monika's character is WRITTEN as pure, so your reasoning completely destroys her original character and makes no sense. Again, real world people don't completely apply and just because that may be common doesn't make it a good thing or right for that matter.

> I mean just mention sayyyy Whether u are dominant or submissive. 

As for this one, who even asks that? And how the hell do you expect people who've never been in a relationship to even answer that? It just seems very unnecessary to me.

> the fact that females do have sexual emotions too
> Monika herself should be openly talk about sexuality to the player since obv ur in a relationship with her.

This raises a question. Have you even experienced most of the mod? Have you made it to the 6 month anniversary? Have you gotten 400+ affection? If you haven't, why are you making suggestions for a large character change? There are 2 topics already that talk about sexual things, well kinda 3. One where she says ''I'm a pure and innocent highschool girl.'', one where she talks about masturbation, and one where she asks if you've been looking at porn of her. Did you know that she's a bit more open about those last 2 after 6 months (or maybe it was 400+ affection)? I don't see what else could be added about sexual topics that wouldn't go too far beyond these ones.

I think that's all I wanted to say.

**Comment by MaximumRustler** [comment]
And yes, I'm a bit triggered by your comments.

**Comment by Torwald39** [comment]
I have maybe 2 suggestions for conversations : 
- what Monika thinks about porn that pollutes the Internet, a real cancer. 
- Monika has already talked about paraphilia with the fetish on panties, we could then see what she thinks about other weird fetishes found on the net.

**Comment by MaximumRustler** [comment]
Well, for the paraphilia topic she does mention that it's a bunch of things and seemed kinda disgusted by it, and I assume it would be the same for weird/cancerous type of porn. Or at least she's weirded out by things like that since paraphilia is less extreme compared to other fetishes. I can't remember, but I think the devs didn't want to get into any more fetishes beyond that because it would get too complicated and would be too many topics to cover all of them. I'm pretty sure there was a discussion about it a while back and that's what they decided. The devs could answer that better though.

**Comment by MaximumRustler** [comment]
I wouldn't really mind her asking about the player's age. But yeah, I still disagree about adding anything about this than what we already have. There isn't enough info to go on to write it properly and it would just seem like completely the topic writer's opinion on what she would like, rather than it being built on some hint of character that's already there.

**Comment by AllYourBandwidthBelongtoUs** [comment]
if they add anything of sexual nature i wouldn't bothered by it as i'm an adult and that is all i can say because it is my first time using github

**Comment by MaximumRustler** [comment]
@ThePotatoGuy @Rai99 What are your opinions on this so far?

**Comment by nohumanallow3d** [comment]
> It's not that sexual topics bother me, it's that I don't think they'd fit in with Monika's character. I don't see what could be added right now that wouldn't go too far into the subject compared to the ones that are currently in the mod. To me, this mod is supposed to be a representation of Monika's character, not some combination of her and the way common relationships are. I guess the reason I'm defending my points so much is that there are people who take this mod very seriously (like me) and seeing things that don't make sense with her character feel almost like a personal attack against her (not the best example but it'll do). I mean, sure there are some discrepancies, but those are being worked on. I'd rather not see this mod become a thing just to please everyone, as opposed to being a good representation of Monika.

You're right about the part that too much lewdness doesnt really fit with monika  but she does sometimes mention perverted things like allowing you to masturbate if u are thinking of her and i remember there was something she said about her asking you whether you wanted to see her wearing 'sensual stockings' or something

**Comment by nohumanallow3d** [comment]
> Considering her labeling herself as pure and innocent, and the way her life has been up to this point I'm going to wager not even Monika knows herself what her sexual preferences are, she accepts the player regardless of their gender and only the assumption that they're male at first because of the association with MC and the general audience of romance games but that's mostly because of their role in her life and her devotion to them as opposed to anything sexual.
> 
> Either way right now there's no verifying the players age yet so writing a topic assuming that they're 18+ isn't a great idea, it's also a bit creepy to go asking for the players age, feels a bit like a hidden survey. 🤔

yes thats kinda tru since the only age warning so not for under 13s......sexually suggestive content would be ok in my eyes tho since most teenage rated media contain some sort of suggestions

**Comment by Rai99** [comment]
A lot of the time, when people see an issue like this or something about mature content, it's usually seen as somebody wanting to add porn or some serious ecchi. I don't think either of those two things belong in the mod, seeing that Monika is in fact an innocent highschool girl and shouldn't be into anything lewd with the player ~~yet~~ and probably shouldn't be for a long time until we possibly have better systems in place such as affection, morality, etc.

Even if we wanted Monika to have a preference about lewd things, I don't think we have any definitive evidence that would point us in the right direction. Monika would probably be comfortable with anything as long as the player is also comfortable and as long as she's been with the player for a good amount of time. ~~Ideally, much, much more than 3 days.~~ 

**Having Monika _discuss_ these _kinds_ of topics is different I think.** In any relationship, it's important to be open about these kinds of things, even fetishes and fantasies. I think the problem would be bringing it up smoothly. It would probably have to have a bit more context and a smoother transition than "Hey, do you have any fetishes or fantasies?". I tried to make the virginity topic in #2694, but I left it since I couldn't find a good way to incorporate it into the mod without it being too random and out of nowhere. 

The monika_panties topic was recently updated in #2703 to let the player tell Monika if they're into that kind of thing ~~although I find it an interesting choice to add pm variables for it~~. In this case, the option was added onto an already existing topic, so there's context and the topic makes sense. There are guidelines to avoid lewd content, so the priority and acceptance of the change might be surprising and I wouldn't exactly count on an influx of moderately lewd content, especially when made by lesser known sources. 

There might also be some larger concerns that come with verifying the player's age and minors seeing lewd or explicit content. I'm pretty sure it would be pretty easy for anyone to find the mod and lie about their age.

------
tldr:
- Adding porn or ecchi is probably an easy and simple no. 
- It would be hard for Monika to have a firm stance on anything lewd with the player since there isn't much to go off of with that. If anything, Monika would probably be comfortable with anything as long as the player is and the relationship isn't new.
- It might be possible to have topics **discussing** these kinds of things, but probably not fantasizing about them, and the topic would have to have context. ~~Unless community branch~~

**Comment by MaximumRustler** [comment]
Thank you for the response Rai99. These are some of the points I was trying to get across, but again, maybe I was a bit too harsh. It is actually a bit surprising that the monika_panties topic change was accepted, but I'm glad it was a modification to that topic and not completely out of context. If any topics were added about discussing this, not fantasizing about it, I think it should only be available **at least** after the 6 month anniversary (maybe more, depends) because that's when she's more open to the other topics that already exist.

**Comment by nohumanallow3d** [comment]
> A lot of the time, when people see an issue like this or something about mature content, it's usually seen as somebody wanting to add porn or some serious ecchi. I don't think either of those two things belong in the mod, seeing that Monika is in fact an innocent highschool girl and shouldn't be into anything lewd with the player yet and probably shouldn't be for a long time until we possibly have better systems in place such as affection, morality, etc.
> 
> Even if we wanted Monika to have a preference about lewd things, I don't think we have any definitive evidence that would point us in the right direction. Monika would probably be comfortable with anything as long as the player is also comfortable and as long as she's been with the player for a good amount of time. Ideally, much, much more than 3 days.
> 
> **Having Monika _discuss_ these _kinds_ of topics is different I think.** In any relationship, it's important to be open about these kinds of things, even fetishes and fantasies. I think the problem would be bringing it up smoothly. It would probably have to have a bit more context and a smoother transition than "Hey, do you have any fetishes or fantasies?". I tried to make the virginity topic in #2694, but I left it since I couldn't find a good way to incorporate it into the mod without it being too random and out of nowhere.
> 
> The monika_panties topic was recently updated in #2703 to let the player tell Monika if they're into that kind of thing although I find it an interesting choice to add pm variables for it. In this case, the option was added onto an already existing topic, so there's context and the topic makes sense. There are guidelines to avoid lewd content, so the priority and acceptance of the change might be surprising and I wouldn't exactly count on an influx of moderately lewd content, especially when made by lesser known sources.
> 
> There might also be some larger concerns that come with verifying the player's age and minors seeing lewd or explicit content. I'm pretty sure it would be pretty easy for anyone to find the mod and lie about their age.
> 
> tldr:
> 
> * Adding porn or ecchi is probably an easy and simple no.
> * It would be hard for Monika to have a firm stance on anything lewd with the player since there isn't much to go off of with that. If anything, Monika would probably be comfortable with anything as long as the player is and the relationship isn't new.
> * It might be possible to have topics **discussing** these kinds of things, but probably not fantasizing about them, and the topic would have to have context. Unless community branch

'_yet_ '

In all honesty i think that you're right. That monika should be more curious than lewd and that it would be better to put a smooth change like you said. Obv porn and cringey-ass ecchi is out of the question (what anime do you watch?Harem?ewwwww). However, I wouldnt think the occasional blush-smirk wouldnt be a bad thing

**Comment by CrocWorks** [comment]
>The monika_panties topic was recently updated

Oh, topics have been getting get updated? Is there some way to alert us that a topic has been updated (kinda like how there's an **Unseen** button for the new topics?

**Comment by Rai99** [comment]
I don't think there is, but personally, I find it best to just let repeat topics run and be pleasantly surprised when you see something new.

**Comment by CrocWorks** [comment]
Oh, these were random topics? I thought they were the topics that were only clicked on in the menu. If they're random, then yeah it's fine to leave it as is.

**Comment by Foxinsunglasses** [comment]
I'm still for more suggestive content knowing people evolve sure she may be innocent now but people change over time. I as a collage student am in almost name alone the same person I was a few years ago. Monika herself addresses this when she talks of her middle school life.

Side note this feels like the council of Nicaea

**Comment by MaximumRustler** [comment]
While it's true that people change over time, I'd say the opposite is just as true. For example I'm 21 and I'm more or less the same as I was at 17, mostly because I had to mature at a earlier age because of certain things that happened in my life. I had a lot of time to myself to sort out what kind of person I am, what I look for in people, and my opinions on various things. I still agree with what Rai99 said, that discussing these topics should come later on, maybe when better systems are in place such as the player's influence on Monika's personality or whatever it was called.

**Comment by MaximumRustler** [comment]
@Foxinsunglasses Also, what's council of Nicaea? I don't know the reference. lol

**Comment by TheGlader2** [comment]
@Foxinsunglasses How does this feel like the council of Nicaea? Haven't these kind of forums been around for a while?

**Comment by AllYourBandwidthBelongtoUs** [comment]
why don't the devs release two updates not at the same but one with sexual topics and one without if that could work that maybe might keep the people that want sexual topics and things happy and keep that don't and  but i don't know how it takes to get an update out but i think it might take weeks or months to make updates or they could release it as an rar or zip file then the people that download can put it in the game itself it is easy to put files but i guess you have to keep kids seeing adult content at young age.

**Comment by Unknownpony** [comment]
What's the point of doing that when there is already sensative mode? Just keep adult topic out of sensative mode or just add additional checkbox like sensative mode but with default it off. Making two builds isn't really good idea.
Beside let's be for real. Every adult/porn content is just behind "are you 18?" stuff. Even steam has only that.

**Comment by Foxinsunglasses** [comment]
@MaximumRustler 
It was when the Christian church decided what was and wasn't official doctrine. Basically separating the fan fiction from the cannon.

**Comment by MaximumRustler** [comment]
@Foxinsunglasses Ahh, got it.

**Comment by Shiro2564** [comment]
yes it would be a nice idea to include that, but there are also kids that
play this game, thats the only problem

**Comment by Unknownpony** [comment]
Sorry but isn't DDLC in itself a 18+ game in itself? Like you
"this game is not suitable for children or those who are easily disturbed"
I am not sure why people asume kids play this mod when main game isn't for kids.

**Comment by Rai99** [comment]
DDLC doesn't say anywhere that it's 18+. According to MASC, MASC "follows DDLC's own policy of no players under the age of 13".

The game warns that it is not suitable for children or those easily disturbed, which is a subjective restriction, especially those who are under 18. 

Even then, how many times does an age rating actually stop someone and how many people never lied about their age? 

Personally for me, I'm not sure why or how people assume age ratings actually stop most people and that age restricted games are free from underage players. In my experience, there are definitely little kids under 10 who play GTA.

**Comment by nohumanallow3d** [comment]
> DDLC doesn't say anywhere that it's 18+. According to MASC, MASC "follows DDLC's own policy of no players under the age of 13".
> 
> The game warns that it is not suitable for children or those easily disturbed, which is a subjective restriction, especially those who are under 18.
> 
> Even then, how many times does an age rating actually stop someone and how many people never lied about their age?
> 
> Personally for me, I'm not sure why or how people assume age ratings actually stop most people and that age restricted games are free from underage players. In my experience, there are definitely little kids under 10 who play GTA.

Thats what im trying to say. It would be okay to add suggestive content since there is a 13+ warning.

**Comment by Shiro2564** [comment]
yes thats true

**Comment by AsuraShun** [comment]
don't actually add this 18+ content, parents will go crazy and try to get this banned if they see kids playing this,even if the content isn't explicit

**Comment by AsuraShun** [comment]
many games that had something even remotely suggestive got censored and the art style redone, these oversensitive parents would only try to ruin the game by tainting its reputation

**Comment by Shiro2564** [comment]
but DDLC says that it isn't for children so if the parents say someting
they need to look at that message

**Comment by AsuraShun** [comment]
still won't stop them from making a ruckus over it

**Comment by nohumanallow3d** [comment]
> still won't stop them from making a ruckus over it

have parents sued porn websites and succeeded tho??

**Comment by ThePotatoGuy** [comment]
h content will always be via a submod (aka not in official MAS). Some of us here may actually make one after MAS is more stable/mature.
